ci-operator/config/open-cluster-management/cluster-backup-operator/open-cluster-management-cluster-backup-operator-main.yaml

```diff
@@ -0,0 +1,81 @@
+base_images:
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+  repo-brancher:
+    name: repo-brancher
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile
+  inputs:
+    open-cluster-management_builder_go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
+  to: cluster-backup-controller
+promotion:
+  name: "2.4"
+  namespace: open-cluster-management
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit-tests
+  commands: make test
+  container:
+    from: src
+- as: sonar
+  commands: |
+    export SONAR_GO_TEST_ARGS="-short ./..."
+    export SELF="make -f /opt/build-harness/Makefile.prow"
+    make -f /opt/build-harness/Makefile.prow sonar/go/prow
+  container:
+    from: src
+  secrets:
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
+- as: pr-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: cluster-backup-controller
+    env:
+      IMAGE_REPO: cluter-backup-controller
+      REGISTRY_ORG: open-cluster-management
+    workflow: ocm-ci-image-mirror
+- as: pr-merge-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: cluster-backup-controller
+    env:
+      IMAGE_REPO: cluster-backup-controller
+      REGISTRY_ORG: open-cluster-management
+    workflow: ocm-ci-image-mirror
+- as: fast-forward
+  commands: |
+    git clone -b master https://github.com/openshift/release.git /tmp/release
+    repo-brancher --fast-forward --confirm \
+        --config-dir=/tmp/release/ci-operator/config/open-cluster-management/cluster-backup-operator/ \
+        --username=acm-cicd \
+        --token-path=/etc/github/token \
+        --current-promotion-namespace=open-cluster-management \
+        --current-release=2.4 \
+        --future-release=2.4
+  container:
+    from: repo-brancher
+  postsubmit: true
+  secrets:
+  - mount_path: /etc/github
+    name: acm-cicd-github
```

Comment:
Fast forward process has changed, but docs have not yet been updated. Use this stanza instead:

```
- as: fast-forward
  postsubmit: true
  steps:
    env:
      DESTINATION_BRANCH: release-2.4
    workflow: ocm-ci-fastforward
```